components/dashboards/thumbnail-list/component.js

```diff
@@ -1,32 +1,69 @@
 import React, { PureComponent } from 'react';
+import classnames from 'classnames';
+
 import PropTypes from 'prop-types';
-import { Link } from 'routes';
 
 // styles
 import './styles.scss';
 
 class DashboardThumbnailList extends PureComponent {
-  static propTypes = { dashboards: PropTypes.array.isRequired }
+  static propTypes = {
+    dashboards: PropTypes.array,
+    onSelect: PropTypes.func,
+    user: PropTypes.bool
+  }
 
-  render() {
-    const { dashboards } = this.props;
+  static defaultProps = {
+    onSelect: () => {},
+    dashboards: [],
+    user: false
+  }
 
+  render() {
+    const { dashboards, onSelect, user } = this.props;
+    const buttonDefaultClass = 'thumbnail-list-item';
+    const buttonClass = user ? buttonDefaultClass : `${buttonDefaultClass} no-user`;
     return (
-      <ul className="c-dashboard-thumbnail-list">
-        {dashboards.map(_dashboard => (
-          <li
-            key={_dashboard.slug}
-            style={{ backgroundImage: `url(${_dashboard.photo.original})` }}
-          >
-            <Link
-              to="dashboards_detail_custom"
-              params={{ slug: _dashboard.slug }}
-            >
-              <a><span>{_dashboard.name}</span></a>
-            </Link>
-          </li>
-        ))}
-      </ul>
+      <div className="c-dashboards-gallery-list">
```

Comment:
Why did you change the name of the CSS class used to a different value that doesn't correspond to the name of the component?
Besides, I've seen that the former CSS class `c-dashboard-thumbnail-list` is no longer used anywhere in the code. 
Why didn't you replace the old CSS code with the one that you put in the new CSS file called `dashboards_gallery.scss`?

Reply:
dashboards_gallery renamed to c-dashboards-thumbnail-list. fixed here: https://github.com/resource-watch/resource-watch/pull/1185/commits/906f1bce4b882a0b0bfb84a641261c884020b757